desktop/src/com/mygdx/game/ServerLauncher.java

```diff
@@ -0,0 +1,21 @@
+package com.mygdx.game;
+
+import io.vertx.core.Vertx;
```

Comment:
Vertx?? A nie mieliśmy używać websocketów z libgdx? Czemu taka zmiana? Używałęś tego kiedyś?

Reply:
W tym linku gdzieś tam są example i jest napisane ze websockety z libgdx nie wspierają serwerów tylko klienta, i skorzystali właśnie z tego

Reply:
aaaa, to dobrze.

To fajnie, że korzystam z kolejnych technologii. Przynajmniej ta inżynierka będzie bardziej rozwijająca.

Reply:
Ogólnie to działa tak ten przykład, że klient się łączy wysyła coś, a serwer po 5 sekundach go rozłącza :P ale działa, jak chcesz możesz sie pobawic i popatrzeć jak działa, pomyślę jak to przenieść do tego naszego serwera